web/src/features/charts/elements/AreaGraph.tsx

```diff
@@ -142,33 +142,33 @@ function AreaGraph({
     () => getValueScale(containerHeight, totalValues),
     [containerHeight, totalValues]
   );
-  const startTime = datetimes?.at(0);
-  const lastTime = datetimes?.at(-1);
-  const interval = datetimes?.at(-2);
+  const startTime = datetimes.at(0);
+  const lastTime = datetimes.at(-1);
+  const interval = datetimes.at(-2);
 
-  if (!startTime || !lastTime || !interval) {
-    return null;
-  }
```

Comment:
There is no point in moving this one as all the values would have been used by the time it goes through all the hooks.

Reply:
They are used in other hook calls which means that by the point we would exit early if we moved it all the values would have already been used making the check if they are fall pointless.